app/assets/stylesheets/global/_base.scss

```diff
@@ -159,16 +167,15 @@ video {
 
 .v2-banner {
   background-color: $color_bg-dark;
-  position: relative;
-  padding-top: 2em;
+  background-size: 40%;
   color: $color_font-light;
   font-size: 20px;
   font-weight: normal;
-  background: $color_bg-dark;
-  background-size: 40%;
+  padding-top: 2em;
```

Comment:
Is this supposed to be `rem` or `em`? Totally fine if it's the latter, I just haven't seen it much in our repos.

Reply:
Typo, good catch.